gobblin-core/src/main/java/gobblin/converter/objectstore/ObjectStoreConverter.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+package gobblin.converter.objectstore;
+
+import gobblin.configuration.WorkUnitState;
+import gobblin.converter.Converter;
+import gobblin.converter.SchemaConversionException;
+import gobblin.writer.objectstore.ObjectStoreOperation;
+
+/**
+ * A converter of {@link ObjectStoreOperation}s. The output record of any subclasses is of type {@link ObjectStoreOperation}
+ *
+ * @param <SI> Type of input record schema
+ * @param <DI> Tnput record type
+ */
+public abstract class ObjectStoreConverter<SI, DI> extends Converter<SI, SI, DI, ObjectStoreOperation> {
```

Comment:
`public SI convertSchema` does't really make sense for the object store use case. This class exists to that every operation converter don't need to implement `public SI convertSchema`